thirdparty/lua-rapidjson/flags.patch

```diff
@@ -0,0 +1,18 @@
+diff -uNr a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt	2020-06-07 17:18:56.373394505 +0200
++++ b/CMakeLists.txt	2020-06-07 21:42:18.129006112 +0200
+@@ -31,8 +31,12 @@
+         set(PLAT "linux")
+         set(LINK_FLAGS "-shared")
+     endif(APPLE)
+-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -Wall -fPIC -march=native")
+-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -Wall -fPIC -march=native")
++    add_compile_options(-g -Wall -fPIC)
```

Comment:
It was already merged so we can get rid of it. :-)

Reply:
Sorry, that's not quite correct as I wrote in https://github.com/koreader/koreader-base/pull/1113#issuecomment-640538237

-march=native isn't suitable except for compiling things for yourself on your own PC. Hence why we do:

https://github.com/koreader/koreader-base/blob/9f3f0093267b75da1d3772a006d0138124544fc2/Makefile.defs#L441-L449